index.html

```diff
@@ -75,6 +75,16 @@ <h2>Autofill</h2>
 	<li><a href="./autofill/modal.html">Email autofill form within self-closing modal</a></li>
 	<li><a href="./autofill/form-submission.html">Form submission detection and autofill</a></li>
 	<li><a href="./autofill/frame-form-submission-parent.html">Form submission detection and autofill within an iframe</a></li>
+  <li>
+		Autoprompt
```

Comment:
Can we expand a bit more what autoprompt is? I'm guessing something like "automatically prompt to store credentials after login form is detected"?